lib/scrabble.rb

```diff
@@ -1,18 +1,43 @@
 class Scrabble
 
+  attr_reader :tally
+
+  def initialize
+    @tally = 0
+  end
+
   def score(word)
-    1
+    if word == nil
+      nil_score
+    else
+      score_calculation(word)
+    end
+    return tally
+  end
+
+  def score_calculation(word)
+    word.to_s.each_char do |letter|
+      @tally = tally + point_values[letter.upcase]
+    end
+  end
+
+  def nil_score
+    return tally = 0
```

Comment:
Like that you're breaking up these steps into smaller methods. Since you're returning `tally` on 15, consider using this method to just set the instance variable to 0.

Currently this works, but it's because you're setting a **local** variable to 0, and assignment returns the value that was assigned.